commands/ask.py

```diff
@@ -14,11 +14,11 @@
 
 
 async def on_ask(author: Member, guild: Guild, database: Database, channel: Union[TextChannel, Thread],
-                 text: str, openai_client: OpenAiClient, message: Optional[Message],
+                 text: str, ai_client: AiClient, message: Optional[Message],
```

Comment:
This variable could be "client", as there's only one client needed in this function
```suggestion
                 text: str, client: AiClient, message: Optional[Message],
```